### ISSUE TITLE
Bugzilla refresher depends on offset, want it to depend on datetime instead

### ISSUE THREAD

**Issue by anthonyjlau** [sailuh/kaiaulu#300]
Using offset is a bit confusing but more importantly it is different from how other parses work. The Bugzilla parser should change the datetime in the API call instead of the offset.

**Comment by carlosparadis** [comment]
@anthonyjlau Did you manually test the created date to see if it would work?

**Comment by anthonyjlau** [comment]
Yes, when I tested changing the created date manually, it works.

For example, [in this API call](https://bugzilla.redhat.com/rest/bug?creation_time=2024-04-26T12:00:00Z&include_fields=_default,comments&limit=20) (https://bugzilla.redhat.com/rest/bug?creation_time=2024-04-26T12:00:00Z&include_fields=_default,comments&limit=20), the `creation_time` starts at 2024-04-26T12:00:00Z. The last issue on the page has a `creation_time` of 2024-04-26T13:29:18. 

So, I change the `creation_time` field to be 2024-04-26T13:29:19 (one more second than the last creation time) and the API call looks like [this](https://bugzilla.redhat.com/rest/bug?creation_time=2024-04-26T13:29:19Z&include_fields=_default,comments&limit=20) (https://bugzilla.redhat.com/rest/bug?creation_time=2024-04-26T13:29:19Z&include_fields=_default,comments&limit=20). 

This will return the same thing as [this API call](https://bugzilla.redhat.com/rest/bug?creation_time=2024-04-26T12:00:00Z&include_fields=_default,comments&limit=20&offset=20) (https://bugzilla.redhat.com/rest/bug?creation_time=2024-04-26T12:00:00Z&include_fields=_default,comments&limit=20&offset=20), which is what is being used currently.
